package.json

```diff
@@ -71,6 +70,7 @@
     "svg-url-loader": "^2.3.0",
     "url-loader": "^0.6.2",
     "uswds": "1.4.2",
+    "vagov-eslint": "file:./packages/vagov-eslint",
```

Comment:
If we decide to publish this package, I think we can move this import into each one of the packages and `lerna` will install it correctly.